models.lua

```diff
@@ -1,2 +1,49 @@
-local autoload = require("lapis.util").autoload
-return autoload("models")
+-- Database abstractions
+
+package.loaded.Users = package.loaded.Model:extend('users', {
+    primary_key = { 'username' },
+    relations = {
+        { 'collections', has_many = 'Collections', key = 'creator_id' }
+    }
+})
+
+package.loaded.Projects = package.loaded.Model:extend('projects', {
```

Comment:
Note: we need to fix this.

Reply:
Hmm. What needs fixing? :thinking:

Reply:
Needs to be `active_projects`.

Reply:
Oh, indeed. I fixed that when merging :)